projectforum/projects/tests/test_projects.py

```diff
@@ -226,39 +225,36 @@ def test_list_projects_by_title_down(self):
     def test_list_projects_by_title_up(self):
         test_create_projects.create_many_projects()
         response = project_filters.get_project_list(status=1,
+                                                    keywords='',
                                                     order='title',
-                                                    salary='Lump',
-                                                    ascending=True,
-                                                    starting_from=1,
-                                                    ending_at=10)
+                                                    salary='lump',
+                                                    ascending=True)
         contents = json.loads(response.content)
         self.assertEqual(1, contents['status'])
         self.assertEqual('A', contents['projects'][0]['title'])
         self.assertEqual('F', contents['projects'][-1]['title'])
         self.assertEqual('D', contents['projects'][3]['title'])
 
-    def test_list_projects_by_created_up(self):
+    def test_list_projects_by_newest(self):
         test_create_projects.create_many_projects()
         response = project_filters.get_project_list(status=1,
+                                                    keywords='',
                                                     order='timestamp',
-                                                    salary='Lump',
-                                                    ascending=True,
-                                                    starting_from=1,
-                                                    ending_at=10)
+                                                    salary='lump',
+                                                    ascending=False)
         contents = json.loads(response.content)
         self.assertEqual(1, contents['status'])
         self.assertEqual('B', contents['projects'][0]['title'])
```

Comment:
B is actually the oldest, not the newest. The ascending=False setting is right I think so I'm not sure what's up.